pymodbus/datastore/context.py

```diff
@@ -27,6 +28,43 @@ def decode(self, fx):
         """
         return self._fx_mapper[fx]
 
+    async def async_getValues(self, fc_as_hex, address, count=1):
```

Comment:
new methods as well as changed methods must be typed, as part of our ongoing effort to having the codebase strongly typed.

Reply:
updated typehints to be in place